src/services/ogc-features/src/main/resources/application.yml

```diff
@@ -18,6 +18,10 @@ springdoc:
     #path: swagger-ui/index.html
     try-it-out-enabled: true
 
+#cors:
+#  allowed-origins: "*"
+#  allowed-methods: GET, POST, PUT, DELETE, OPTIONS
```

Comment:
you could have used a yaml array to avoid the split call, and defined the allowedMethods as a String[] or List in the Java code ?

Reply:
It even does auto split with commas when it's set to String[]

Reply:
you could even have tested with a `Set<String>` to avoid having the same value multiple times ?

Reply:
Putting Set<String> involves to cast as String after : `... .allowedMethods(String.valueOf(allowedMethods))...`